packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -621,15 +628,42 @@ class StackViewport extends Viewport implements IStackViewport {
   }
 
   /**
-   * Sets the properties for the viewport on the default actor. Properties include
-   * setting the VOI, inverting the colors and setting the interpolation type, rotation
+   * Update the default properties of the viewport and add properties by imageId if specified
+   * @param ViewportProperties - The properties to set
+   * @param imageIdIndex If given, we set the default properties only for this image index
+   */
+  public setDefaultProperties(
+    ViewportProperties: StackViewportProperties = {},
+    imageIdIndex?: number
+  ): void {
+    if (imageIdIndex == undefined) {
+      this.globalDefaultProperties = ViewportProperties;
+    } else {
+      this.perImageIdIndexDefaultProperties.set(
+        imageIdIndex,
+        ViewportProperties
+      );
+
+      //If the viewport is one the same imageIdIndex, we need to update the viewport
+      if (this.getCurrentImageIdIndex() == imageIdIndex) {
+        this.setProperties(ViewportProperties);
+      }
+    }
+  }
+
+  /**
+   * Sets the properties for the viewport on the default actor.
+   * and if setProperties is called for the first time, the properties will become default properties for all imageIds
+   * Properties include setting the VOI, inverting the colors and
+   * setting the interpolation type, rotation.
    * @param voiRange - Sets the lower and upper voi
    * @param invert - Inverts the colors
    * @param interpolationType - Changes the interpolation type (1:linear, 0: nearest)
    * @param rotation - image rotation in degrees
    */
```

Comment:
can we change the doc here?

/**
 * Configures the properties of the viewport.
 * This method allows customization of the viewport by setting attributes like
 * VOI (Value of Interest), color inversion, interpolation type, and image rotation.
 * If `setProperties` is called for the first time, the provided properties will
 * become the default settings for all images in the stack in case the resetPropertiese need to be called
 *
 * @param properties - An object containing the properties to be set.
 * @param properties.colormap - Specifies the colormap for the viewport.
 * @param properties.voiRange - Defines the lower and upper Value of Interest (VOI) to be applied.
 * @param properties.VOILUTFunction - Function to handle the application of a lookup table (LUT) to the VOI.
 * @param properties.invert - A boolean value to toggle color inversion (true: inverted, false: not inverted).
 * @param properties.interpolationType - Determines the interpolation method to be used (1: linear, 0: nearest-neighbor).
 * @param properties.rotation - Specifies the image rotation angle in degrees.
 * @param suppressEvents - A boolean value to control event suppression. If true, the related events will not be triggered. Default is false.
 */